modules/ROOT/pages/offline-policy-task.adoc

```diff
@@ -3,10 +3,23 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-An offline policy is applied directly to the runtime, not applied through API Manager. This procedure describes how to apply an offline policy to an API. You create a file in the offline-policies directory. Mule detects the presence of the file and starts applying the policy. Mule generates JSON and XML policy files in a new directory in `%MULE_HOME%/policies`.
+An offline policy is applied directly to the runtime and not through API Manager. This procedure describes how to apply an offline policy to an API. You create a file in the offline-policies directory. 
```

Comment:
The procedure should have steps:

1. Create a file in the offline-policies directory. 
2. Mule detects the presence of the file and starts applying the policy.
3. Mule generates JSON and XML policy files in a new directory in `%MULE_HOME%/policies`

Or, if you don't call it a procedure:

An offline policy is applied directly to the runtime and not through API Manager. To apply an offline policy to an API, create a file in the offline-policies directory. 

Mule detects the presence of the file and starts applying the policy. Mule then generates JSON and XML policy files in a new directory in `%MULE_HOME%/policies`.

Reply:
Did not add in steps but fixed as I deemed fit. This topic will go for a Kim edit later.